src/shared/components/show-hide/ShowHide.scss

```diff
@@ -1,15 +1,15 @@
 @import 'src/styles/common';
 
 .showHide {
-  // increase specificity to be sure to override chevron's own styles 
+  // increase specificity to be sure to override chevron's own styles
   .chevron {
     margin-left: 10px;
-    height: 6px;
+    --chevron-height: 6px;
   }
-}
 
-.label {
-  color: $blue;
-  cursor: pointer;
-  white-space: nowrap;
+  .label {
+    color: var(--show-hide-label-color, $blue);
+    cursor: var(--show-hide-label-cursor, pointer);
```

Comment:
Is `cursor` something that needs configuring? In `ShowHide.tsx`, doesn't a click on the label (if the label exist) always toggle the show/hide?